erizo/generateProject.sh

```diff
@@ -10,13 +10,11 @@ BASE_BIN_DIR="build"
 generateVersion() {
   echo "generating $1"
   BIN_DIR="$BASE_BIN_DIR/$1"
-  if [ -d $BIN_DIR ]; then
-    cd $BIN_DIR
-  else
+  if [ ! -d $BIN_DIR ]; then
```

Comment:
what's the issue with the previous way?

Reply:
in this way cmake is called only the first time that the folder is created after a cleaonOject is called.
I noticed that if cmake is run every time generateProject is called, it build the enteire project every time instead of only the changes made in cpp.

Reply:
Just run buildProject.sh if you implement changes inside existing cpp files, or run generateProject.sh if you add/remove files or dependencies. With this change it won't regenerate cmake structure if the directory already exists.